samples/dialog_service_connector/start_keyword_listening.go

```diff
@@ -1,7 +1,7 @@
-//
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
-//
+
+// Package dialog_service_connector sample for keyword recognition

```

Comment:
This should probably not be here?

Reply:
I'll create a standalone doc.go with this package name and combine the descriptors